option.go

```diff
@@ -84,3 +84,37 @@ func MasterKeyIndicator(mki []byte) ContextOption {
 		return nil
 	}
 }
+
```

Comment:
Would it be possible to not have these options, and have everything configured just from Cipher choice?

I am just starting to read the PR, maybe I will understand better as I go :)

Reply:
Nope. MIKEY (RFC 3830) and `a=crypto` SDP attribute (RFC 4568) allows to disable encryption for SRTP and SRTCP independently. I also would have to create 3 additional crypto profiles for every cipher.

Maybe down the road it would be better to change ProtectionProfile to struct and put various additional params there, and pass it to `CreateContext` instead of options like it is implemented now.